docs/source/async_api.rst

```diff
@@ -356,7 +354,7 @@ Closing a driver will immediately shut down all connections in the pool.
             ``parameters_``.
         :type kwargs: typing.Any
 
-        :returns: the result of the ``result_transformer``
+        :returns: the result of the ``result_transformer_``
```

Comment:
Is a python convention?

Reply:
That's what the parameter is called that takes the result transformer. So I decided to use the name of the parameter. Alternatively just plain text "result transformer" (without `code` formatting) would work.